tree-sitter.json

```
@@ -0,0 +1,64 @@
+{
+  "grammars": [
+    {
+      "name": "php",
+      "camelcase": "PHP",
+      "scope": "source.php",
+      "path": "php",
+      "external-files": "common/scanner.h",
+      "file-types": [
+        "php"
+      ],
+      "highlights": "queries/highlights.scm",
+      "injections": [
+        "queries/injections.scm",
+        "queries/injections-text.scm"
+      ],
+      "tags": "queries/tags.scm"
+    },
+    {
+      "name": "php_only",
+      "camelcase": "PHP_Only",
+      "scope": "source.php",
+      "path": "php_only",
+      "external-files": "common/scanner.h"
```

@amaanq,

This should have queries too right?

```suggestion
      "external-files": "common/scanner.h",
            "file-types": [
        "php"
      ],
      "highlights": "queries/highlights.scm",
      "injections": "queries/injections.scm",
      "tags": "queries/tags.scm"
```

---

yes, good catch